ragna/ui/resources/upload.js

```diff
@@ -0,0 +1,51 @@
+
+
+function upload(files, token, informationsEndpoint, final_callback) {
+    var uploaded_documents = [] 
+
+    Array.from(files).map(file => {
+
+        getUploadInformations(file, token, informationsEndpoint).then(
+
+                function(uploadInformations) {
+                        uploadFile(uploadInformations.url, uploadInformations.data, file)
+                        uploaded_documents.push(uploadInformations.document)
```

Comment:
This originally was filled by the response of `uploadFile`. However, there is no protocol on what this should return. Our built-in endpoint indeed returns the document information. But S3 for example doesn't. Thus, we need to use `uploadInformations`.